roles/network_plugin/cilium/templates/cilium/cbgpa.yml.j2

```diff
@@ -0,0 +1,12 @@
+{% for cbgpadvert in cilium_bgp_advertisements %}
+---
```

Comment:
Can the names of these files not be abbreviated? Just for readability.

Reply:
yeah sure, updated there

Reply:
It seems that the file duplicates with `cilium-bgp-advertisement.yml.j2`,  Do we need to remove this?

Reply:
Ah sorry looks like a bad merge and or squash, will sort now and remove the old templates

Reply:
sorted there, tested again on my side without issue

Reply:
thanks, looks good now

/lgtm

Reply:
/cc @yankay please for the approval